ethereum/rpc/eth_api.go

```diff
@@ -129,20 +129,31 @@ func (e *PublicEthAPI) Syncing() (interface{}, error) {
 }
 
 // Coinbase is the address that staking rewards will be send to (alias for Etherbase).
-func (e *PublicEthAPI) Coinbase() (common.Address, error) {
+func (e *PublicEthAPI) Coinbase() (string, error) {
 	e.logger.Debugln("eth_coinbase")
 
 	node, err := e.clientCtx.GetNode()
 	if err != nil {
-		return common.Address{}, err
+		return "", err
 	}
 
 	status, err := node.Status(e.ctx)
 	if err != nil {
-		return common.Address{}, err
+		return "", err
 	}
 
-	return common.BytesToAddress(status.ValidatorInfo.Address.Bytes()), nil
+	req := &evmtypes.QueryValidatorAccountRequest{
+		ValAddress: common.Bytes2Hex(status.ValidatorInfo.Address),
```

Comment:
I think I'd prefer to use the bech32 consensus address here instead of the hex value
```suggestion
		ValAddress: sdk.ConsAddress(status.ValidatorInfo.Address).String(),
```

Reply:
sure, fixed